terraform/examples/aws/nat-gateway/main.tf

```diff
@@ -0,0 +1,138 @@
+module "firezone_gateway_example" {
+  # Required inputs
+  source         = "github.com/firezone/firezone/terraform/modules/aws/nat-gateway"
+  firezone_token = "YOUR_FIREZONE_TOKEN"
+  base_ami       = "ami-0a640b520696dc6a8" # Ubuntu 22.04
```

Comment:
We should use something like this here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami_ids

Not all AMI IDs are the same for all the regions, plus this AMI will never be updated and we'll have manually do that. It's better if we just select latest version we support for a given region.

Reply:
Fixed